packages/@sfx/core/test/unit/common/core.test.ts

```diff
@@ -113,5 +204,14 @@ describe('Core', () => {
         sinon.match.same(directory)
       );
     });
+
+    it('should clear dependency graph', () => {
+      const unregisterPlugins = stub(CoreUtils, 'unregisterPlugins');
+      core.dependencyGraph = { a: ['a'] };
+
+      core.unregisterAll();
+
+      expect(core.dependencyGraph).to.be.empty;
```

Comment:
I think I like this. No need to stub an internal detail like a call to `createDependencyGraph()`. Unless, that is, we expect an 'empty' dependency graph to be created with more information than an empty object.